Packs/VaronisDataSecurityPlatform/Integrations/VaronisDataSecurityPlatform/VaronisDataSecurityPlatform.yml

```diff
@@ -318,7 +318,7 @@ script:
       required: true
       secret: false
     - default: false
-      description: Maximum number of alerts to retrieve (up to 5k)
+      description: Maximum number of alerts to retrieve (up to 100)
```

Comment:
It is still up to 5k, but the default number is 100 now. Could you please revert